src/views/LatticeView.vue

```diff
@@ -0,0 +1,52 @@
+<script setup lang="ts">
+import { computed, ref } from "vue";
+import type { Scale } from "scale-workshop-core";
+import { mmod } from "xen-dev-utils";
+import ScaleLattice from "@/components/ScaleLattice.vue";
+
+// See ji-lattice-2 branch for ideas and how to extract coordinates from the scale.
+const props = defineProps<{
+  scale: Scale;
+  baseMidiNote: number;
+  keyColors: string[];
+  heldNotes: Map<number, number>;
+}>();
+
+// Please make responsive or something
+const canvasWidth = ref(300);
+const canvasHeight = ref(200);
+
+// Use these to light up degrees of the scale in the lattice
+const heldScaleDegrees = computed(() => {
```

Comment:
This computed property is broken but the intended functionality of lighting up the lattice nodes is desirable. I can fix the property tomorrow...